data/json/npcs/epilogues.json

```
@@ -311,5 +461,155 @@
     "group": "female",
     "id": "npc_female_26",
     "text": "     Was detained and incarcerated by a police robot on multiple counts of looting and vandalism that were tracked and recorded by the few remaining security systems.  Trapped in a cell awaiting police review, she spent weeks crying out for help and slowly consuming what supplies she carried.  Two weeks after being dragged to her cell she died from dehydration."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_27",
+    "text": "     Shot by the Old Guard a few weeks later, hunted down after she robbed an important caravan."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_28",
+    "text": "     Travelled north, she eventually found a quiet community there and spent the rest of her life farming.  She's survived by four daughters."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_29",
+    "text": "     She became a runner for the Old Guard and was eventually permitted to join.  She spent the rest of her life hunting raiders.  She died in a skirmish after a few years, sacrificing her life to let her unit retreat.  Her name is venerated in songs and inscriptions."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_30",
+    "text": "     Shortly after your death she joined the Free Merchants, becoming a caravan guard and after a few months became the new liaison in the Refugee Center.  She died a few years later of a mysterious ilness."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_31",
+    "text": "     She was found half-dead by the Old Guard who took her in.  She became a famous scavenger, known for finding an intact stash of antibiotics."
```

I don't feel like that find is emphasized enough to seem important - players find stashes of antibiotics fairly often. Perhaps something like 
```suggestion
    "text": "     She was found half-dead by the Old Guard who took her in.  She became a famous scavenger, known for finding a large intact stash of antibiotics."
```

---

I agree, it was supposed to say "experimental antibiotics", must have deleted the word somehow when editing.